### ISSUE TITLE
Add IL code viewing for C#

### ISSUE THREAD

**Issue by isnakolah** [dotnet/interactive#1854]
Create functionality to view the IL code for C# code in block

**Comment by jonsequitur** [comment]
We have this functionality in the ExtensionLab package. We'll probably break this into its own package at some point. Give it a try and let us know what you think.

<img width="938" alt="image" src="https://user-images.githubusercontent.com/547415/151018326-b49f6cae-a08a-462d-9f6b-d209a51c553b.png">

@vzarytovskii

**Comment by vzarytovskii** [comment]
> We have this functionality in the ExtensionLab package. We'll probably break this into its own package at some point. Give it a try and let us know what you think.
> 
> <img width="938" alt="image" src="https://user-images.githubusercontent.com/547415/151018326-b49f6cae-a08a-462d-9f6b-d209a51c553b.png">
> 
> @vzarytovskii 
> 
> 
> 

Yeah, I'm planning to do some improvements + have it as a separate packages.
